scripts/uninstall-aica-docker.sh

```diff
@@ -3,5 +3,6 @@
 SYMLINK=/usr/local/bin/aica-docker
 if test -f "$SYMLINK"; then
```

Comment:
I feel this was the line that was not working. As it did not print the echo. So I would prefer that we remove the if and print the echo line even if the file is not here. Otherwise, here, it would actually delete the file on my side but do not print the echo which I think is worse.

Reply:
Confirmed that removing the `if` statement solved my issue